paper/src/main/java/com/denizenscript/denizen/paper/utilities/BlockTagsSetter.java

```diff
@@ -0,0 +1,98 @@
+package com.denizenscript.denizen.paper.utilities;
+
+import com.denizenscript.denizen.Denizen;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizen.utilities.VanillaTagHelper;
+import com.denizenscript.denizencore.utilities.ReflectionHelper;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.destroystokyo.paper.event.server.ServerTickEndEvent;
+import io.papermc.paper.plugin.bootstrap.BootstrapContext;
+import io.papermc.paper.plugin.configuration.PluginMeta;
+import io.papermc.paper.plugin.lifecycle.event.types.LifecycleEvents;
+import io.papermc.paper.registry.RegistryKey;
+import io.papermc.paper.registry.TypedKey;
+import io.papermc.paper.registry.tag.TagKey;
+import net.kyori.adventure.text.logger.slf4j.ComponentLogger;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.block.BlockType;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.plugin.java.JavaPlugin;
+
+import java.io.File;
+import java.lang.invoke.MethodHandle;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class BlockTagsSetter implements Listener {
+
+    public static final MethodHandle JAVA_PLUGIN_GET_FILE = ReflectionHelper.getMethodHandle(JavaPlugin.class, "getFile");
```

Comment:
reflection for this seems overcomplicated, we own the relevant plugin instance, should be no trouble calling methods the regular way on it

Reply:
The method was `protected`, but yeah can just override it in `Denizen`, my bad